CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
 ## NEXT
-
+* Update to latest WARDEN-roboto, bringing Google's PKI cert path validator to guard against cert path validations
+* Per-App trust anchor overrides
+* BEHAVIOURAL CHANGE:
+  * Android attestation errors due to certificate revocation don't fall into the `Content` exception category any more
+  * Instead, they are now more correctly binned into the `Certificate.Trust` exception subtree
```

Comment:
Is there a test for this?

Reply:
turns out: i cannot commit those tests, because they are from real devices (which I don't have access to) that run in a production environment, so we are not allowed to publish them here